modules/ROOT/pages/external-org-access.adoc

```diff
@@ -0,0 +1,41 @@
+= External Organization Access
+ifndef::env-site,env-github[]
+include::_attributes.adoc[]
+endif::[]
+
+Organizations that have their own, separate domains in Anypoint Platform can share resources with one another using external access lists of trusted organizations.
```

Comment:
We should double check (with marketing?) if we're allowed to say "trusted" organizations, since "trust" is a special word in the Salesforce world.

Reply:
Actually, I don't think we are permitted to do so. Might be good to think of an alternative regardless.

Reply:
Maybe just "Organizations that have their own, separate domains in Anypoint Platform can share resources with one another using external access lists."

Reply:
```suggestion
Organizations that have their own, separate domains in Anypoint Platform can share resources with one another using external access lists.
```